frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -92,6 +93,10 @@ export function AnsibleAutomationsForm(props: {
     const { t } = useTranslation(['cluster', 'common', 'credentials', 'create'])
     const { ansibleCredentials, clusterCurator, isEditing, isViewing } = props
 
+    const [featureGates] = useRecoilState(featureGatesState)
+    const automationFeatureGateFlag =
+        featureGates.filter((featureGate) => featureGate.spec?.featureSet === 'AutomationStepsAllEnable').length > 0
+
```

Comment:
You can just use a `.find` here, I think we can just move it down to where it's referenced, instead of making a new variable that we have to keep track of

Reply:
and let's reference the resource name too, instead of the featureSet value